### ISSUE TITLE
Shared Lib option does not work for MSVC build

### ISSUE THREAD

**Issue by arth7s3** [semlanik/qtprotobuf#264]
Build with option BUILD_SHARED_LIBS does not work for MSVC build. Build fails with multiple errors.
Cannot build project DLLs, the only available option for qmake projects (since cannot statically integrate the project).
Works fine on linux.

Build the project using MSVC:
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvarsall.bat x64
mkdir build && cd build
cmake .. -DBUILD_SHARED_LIBS=ON
cmake --build . --config DEBUG -- /m:4

output:
Todo - add ouput

**Comment by semlanik** [comment]
Yeah it's a known issue, that's why CI only tests static builds with MSVC. For now all work in this repo is on hold. Will back to fixing this issue later on.
